handlers/assign/assign.go

```diff
@@ -0,0 +1,57 @@
+package assign
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+)
+
+const (
+	Assign   = "/assign"
```

Comment:
We have already merged the assign/unassign functionalities in #2.

Reply:
Yep. I changed the directory of assign/unassign.
since assign/unassign and labels will handle the same IssueCommentEvent.

Reply:
Okay, so do we need the `assign` directory? As we are putting `/unassign` in the same file, what about change `assign.go` to `assign_unassign.go` and remove the `assign` directory if nothing else to be added?

Reply:
I guess it is better to create different directory for the different action/operation.
So it is convenient to read the codes.
What about `assignees` as the directory name?

Reply:
I am open to that. Can we learn something from kubernetes/test-infra?

Reply:
OK, It seems like test-infra is using `assign` :(
https://github.com/kubernetes/test-infra/blob/master/prow/plugins/assign/assign.go

Reply:
Okay, good. I buy it :)

Reply:
💯